code/game/objects/items/weapons/design_disks/NeoTheology.dm

```diff
@@ -330,6 +331,7 @@
 	license = 12
 	designs = list(
 		/datum/design/autolathe/gun/regulator = 3, // "NT SG \"Regulator 1000\""
+		/datum/design/autolathe/gun/regulator/army, = 3, //army version//
```

Comment:
```suggestion
		/datum/design/autolathe/gun/regulator/army = 3, //army version//
```

Reply:
for some reason VSC wont let me compile

Reply:
okay so it was literally just a single comma. amazing